src/utils.js

```diff
@@ -42,7 +43,11 @@ function bytesToHex(a) {
   }).join('');
 }
 
-function createAccountID(pubKeyBytes) {
+function hexToBytes(a) {
+  return (new BN(a, 16)).toArray();
```

Comment:
see comment on the other pull request re: elliptic.utils.toArray. Be aware that it always makes a copy when the value is already an array, which may not always make sense.